Cargo.toml

```diff
@@ -3,7 +3,12 @@ name = "netavark"
 version = "1.0.4-dev"
 edition = "2018"
 authors = ["github.com/containers"]
+license = "Apache-2.0"
+readme = "README.md"
 description = "A container network stack"
+homepage = "https://github.com/containers/netavark"
+repository = "https://github.com/containers/netavark"
+categories = ["command-line-utilities"]
```

Comment:
netavark is not really a command line utility, is there a fixed list with categories? I would just set `containers` and `networking`.

Reply:
Its a suggestion in packaging cli manual page not sure if it really matters. But i see other popular CLI tools sharing this category. Let me add `containers` and `networking` as well.

Reply:
Sure but netavark is not used manually by a person so I say it is not a command line utility.